### PR TITLE
ArraySchema.items: add typeDSL & other cleanups

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: salad/schema_salad/metaschema/metaschema_base.yml [hunk 1]
```diff
@@ -173,4 +173,5 @@ $graph:
         _id: "sld:items"
         _type: "@vocab"
         refScope: 2
+        typeDSL: True
```
Comment (mr-c) [inline salad/schema_salad/metaschema/metaschema_base.yml:176]: This is new as of https://github.com/common-workflow-language/schema_salad/pull/429

Comment (mr-c) [reply salad/schema_salad/metaschema/metaschema_base.yml:176]: @bogdang989 Does Benten or any other SBG code care about the `typeDSL`?

**Comment by mr-c** [comment]
Request from @tetron Please test `cwltool` and cwlutils's python codegen with the test case https://github.com/common-workflow-language/cwl-utils/issues/93#issue-988277083 and report back

File: salad/schema_salad/metaschema/metaschema_base.yml (hunk 1 above)
Comment (bogdang989) [reply salad/schema_salad/metaschema/metaschema_base.yml:176]: Just checked internally, there should be no impact to any sbg code here

Comment (mr-c) [reply salad/schema_salad/metaschema/metaschema_base.yml:176]: Thanks!

**Comment by mr-c** [comment]
I can't reproduce the issue, so I'm closing for now

File: salad/schema_salad/metaschema/salad.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Author:
 
-* Peter Amstutz <peter.amstutz@curii.com>, Arvados Project, Curii Corporation
+* Peter Amstutz <pamstutz@veritasgenetics.com>, Veritas Genetics
```
Comment (tetron) [inline salad/schema_salad/metaschema/salad.md:5]: ???

**Review by tetron — APPROVED** [review]

**Pull request merged**